.github/workflows/ci.yml

```
@@ -37,11 +41,20 @@ jobs:
       - name: Run tests
         run: npm test
   tests-macos:
-    runs-on: macos-latest
     strategy:
       fail-fast: false
       matrix:
-        node-version: [16.x, 18.x, 20.x, 22.x]
+        os: [macos-latest]
+        node-version: [18.x, 20.x, 22.x]
+        python-version: ['3.13']
+        include:
+          - os: macos-latest
+            node-version: 16.x
+            python-version: '3.10'
+          - os: macos-13
```

GitHub Actions will drop `macos-13` in 3 days.
```suggestion
          - os: macos-15-intel
```